pkg/bootstrap/platform/gcp.go

```diff
@@ -33,6 +37,7 @@ const (
 	GCPProjectNumber     = "gcp_project_number"
 	GCPCluster           = "gcp_gke_cluster_name"
 	GCPLocation          = "gcp_location"
+	GCEInstance          = "gcp_gce_instance"
```

Comment:
in adding this, we should update the code in istio/proxy that uses hostname when an instance name is needed.